examples/step-44/step-44.cc

```diff
@@ -2470,6 +2470,17 @@ namespace Step44
     constraints.clear();
     const bool apply_dirichlet_bc = (it_nr == 0);
 
+    // In this particular example, the boundary values will be calculated for
+    // the two first iterations of Newton's algorithm. In general, one would
+    // build non-homogeneous constraints in the first iteration (that is, when
```

Comment:
We refer to this iteration as the `zeroth` iteration previously, so we should do the same here to be consistent.